experiments/README.md

```diff
@@ -154,3 +184,31 @@ experiments/results_cameo/
             |- 7ER0_A/
             ...
 ```
+
+## pLMFold inference
+
+To run inference on the pLMFold model for the set of CAMEO sequences in a FASTA file:
+
+```bash
+python experiments/inference_plmfold.py \
+    -f datasets/sequences_cameo.fasta \
+    -o experiments/inference_results
+```
+
+The predicted structures are written in PDB format to `experiments/inference_results` by default. Note that custom FASTA files can be used instead. These can be created from PDB entries as:
+
+```bash
+curl https://www.rcsb.org/fasta/entry/7ZZ5 \
+    -o datasets/input_sequences.fasta
+```
+
+## Timing/memory estimations
+
+- Training step timings: ~3.5 secs for pLMFold (on TPUs v2-128) and ~14 secs for AlphaFold `model_1_ptm` (on TPUs v3-8).
```

Comment:
This makes it sound like we include gradient accumulation... just say training with 1 sample per TPU core needs a v2 (with 8GB of memory per core) for pLMFold and takes ~3.5 sec/step, compared to a v3 (16GB per core) and ~14 sec/step for AlphaFold.